Xamarin.PropertyEditing.Mac/Resources/LocalizationResources.resx

```diff
@@ -122,4 +122,28 @@
     <value>{0} InputMode Editor</value>
     <comment>Editor for InputMode Value</comment>
   </data>
+  <data name="AccessibilityPathEditor" xml:space="preserve">
+    <value>{0} Path Editor</value>
```

Comment:
This is missing the actual property name itself.

Reply:
Fixed